### DISCUSSION TITLE
Option to limit variations

### DISCUSSION THREAD

**Discussion by choseh** [pretix/pretix#2251] [Q&A] [answered]
Hi everyone,

we're using bundled products to give people the option to add meals to their tickets for individual days (e.g Buffet Friday, Buffet Saturday, Buffet Sunday, etc.). The buffet product is split into variations for the individual days. Since it doesn't make sense to add multiple buffets for a day, it would be nice to limit each variation to 1 max. I couldn't find a way to do that, is it possible to add such functionality?

Thanks,
Alex

**Comment by raphaelm** [comment, marked as answer]
I assume you're talking about add-on products, bot bundled products?

You'd need to use various products in the same category, not variations, sorry.

This might provide some insight:
https://docs.pretix.eu/en/latest/user/events/guides/workshops.html

**Reply by choseh** [reply]
Yup, add-on products - german version here, sorry. :)

**Reply by choseh** [reply]
So, do I get that right that I will have to create the Buffets like regular products and can't use them as add-ons?
